pages/architecture/audits.mdx

```diff
@@ -1,6 +1,23 @@
 # Audits
 
-Our smart contracts featuring the subscription (auto-renewal) functionality have undergone meticulous audits conducted by Nethermind and ChainSecurity, independent third-party security firms. These esteemed entities meticulously reviewed our codebase to assure users of the safety and reliability of utilizing this feature.
+
+Our smart contracts about the identity and naming, then the subscription featurs have been successfully audited by several firms.
+
+
+## Identity and naming contract
+
+
+The identity and naming contracts have been audited by Cairo Security Clan and Subsix.
+
+
+[🔗 Cairo Security Clan Report](https://github.com/starknet-id/naming/blob/testnet/audits/cairo_security_clan.pdf)
+
+[🔗 Subsix Report](https://github.com/starknet-id/naming/blob/testnet/audits/subsix.pdf)
+
+
+## Subscription
+
+The subscription (auto-renewal) functionality has undergone meticulous audits conducted by Nethermind and ChainSecurity, independent third-party security firms. These esteemed entities meticulously reviewed our codebase to assure users of the safety and reliability of utilizing this feature.
```

Comment:
I think we should refactor here to really show the security of our contracts

Here is my proposition using GPT, please check back dates and links to make sure it works and is coherent.

```md
## Security Audits of Starknet ID's Smart Contracts

Starknet ID's smart contracts have undergone rigorous security audits by independent third-party firms, ensuring the highest level of security and reliability. No critical issues were found during these audits.

### Audit Details

1. **ChainSecurity (October 2023)**

   ChainSecurity performed a comprehensive audit of the subscription (auto-renewal) functionality, **and no critical issues were found.**

   [🔗 ChainSecurity Report](https://github.com/starknet-id/auto_renew_contract/blob/main/audits/chainsecurity.pdf) 

2. **Nethermind (October 2023)**

   Nethermind conducted an extensive review of the subscription (auto-renewal) functionality and **no critical issues were found.**

   [🔗 Nethermind Report](https://github.com/starknet-id/auto_renew_contract/blob/main/audits/nethermind.pdf) 

3. **Cairo Security Clan (May 2024)**

   The identity and naming contracts were audited by Cairo Security Clan, **No critical issues were found.**

   [🔗 Cairo Security Clan Report](https://github.com/starknet-id/naming/blob/testnet/audits/cairo_security_clan.pdf)

4. **Subsix (Independent Auditor, May 2024)**

   Subsix, an independent auditor, also conducted a thorough audit of the identity and naming contracts. **No critical issues were found.**

   [🔗 Subsix Report](https://github.com/starknet-id/naming/blob/testnet/audits/subsix.pdf)

```